packages/features/ee/common/components/LicenseRequired.tsx

```diff
@@ -32,7 +32,7 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
   const Component = as || Fragment;
   return (
     <Component {...rest}>
-      {session.data?.hasValidLicense ? (
+      {session.data?.license.valid ? (
```

Comment:
License validity has been moved to its own object together with the key in order to have it at hand for other usages.